background.js

```diff
@@ -74,6 +74,15 @@ chrome.commands.onCommand.addListener(function (command) {
         update_acc = "u/" + account_num
       }
 
+
+      // Google calendar
+      const calendar_regex = "https:\/\/calendar.google.com\/calendar\/b\/[0-9].\*";
```

Comment:
it wouldn't be .\*
The regex should be .*

Reply:
noted
But worked fine